zerver/models/users.py

```diff
@@ -823,6 +823,7 @@ def has_permission(self, policy_name: str, realm: Optional["Realm"] = None) -> b
             "can_delete_any_message_group",
             "can_delete_own_message_group",
             "can_manage_all_groups",
+            "can_move_messages_between_channels_group",
```

Comment:
Maybe at this point it's worth refactoring this to check the keys of `REALM_PERMISSION_GROUP_SETTINGS`, rather than needing to add a line every time we add a new group setting?

Reply:
Added a commit for this change.